oap-server/server-starter/src/main/resources/otel-rules/aws-dynamodb/dynamodb-endpoint.yaml

```diff
@@ -78,8 +78,7 @@ metricsRules:
     exp: amazonaws_com_AWS_DynamoDB_SystemErrors.tagMatch('Operation','GetItem|Scan|Query|BatchGetItem|TransactGetItems')
   - name: endpoint_write_system_errors
     exp: amazonaws_com_AWS_DynamoDB_SystemErrors.tagMatch('Operation','PutItem|UpdateItem|DeleteItem|BatchWriteItem|TransactWriteItems')
-  - name: endpoint_user_errors
```

Comment:
I can see `User Errors` in the `backend-aws-dynamodb-monitoring.md` doc, should it update as well?